test/Confluent.SchemaRegistry.Serdes.IntegrationTests/Tests/AutoRegisterSchemaDisabled.cs

```diff
@@ -63,14 +63,30 @@ public static void AutoRegisterSchemaDisabled(string bootstrapServers, string sc
                 {
                     Assert.Throws<SerializationException>(() =>
                     {
+                        string guidTopic = new Guid().ToString();
                         try
                         {
                             producer
-                                .ProduceAsync(new Guid().ToString(), new Message<string, int> { Key = "test", Value = 112 })
-                                .Wait();
+                                .ProduceAsync(guidTopic, new Message<string, int> { Key = "test", Value = 112 })
+                                .GetAwaiter()
+                                .GetResult();
```

Comment:
note: this change essentially prevents the exception being annoyingly wrapped in `AggregateException`, and `async`/`await` thing.